src/main/java/com/faforever/client/map/MapService.java

```diff
@@ -485,7 +501,7 @@ public void evictCache() {
     }
 
     return taskService.submitTask(task).getFuture()
-        .thenAccept(aVoid -> noCatch(() -> addSkirmishMap(getPathForMap(folderName))));
```

Comment:
What about the old method ? Are those all places where this issue exists

Reply:
At first glance we know exact name of the directory everywhere else.